components/content/src/page.rs

```diff
@@ -515,13 +515,17 @@ Hello world
 +++
 This page use <sup>1.5</sup> and has footnotes, here's one. [^1]
 
+Here's another. [^2]
+
 <!-- more -->
 
-And here's another. [^2]
+And here's another. [^3]
 
 [^1]: This is the first footnote.
 
-[^2]: This is the second footnote."#
+[^2]: This is the secund footnote.
```

Comment:
Is the spelling error in "second" intentional?

Reply:
secund is a word... but it doesn't fit in that sentence. Maybe auto correct?